pa11ycrawler/spiders/edx.py

```diff
@@ -142,7 +142,7 @@ def __init__(
                     all_blocks="true",
                 )
             )
-            self.start_urls = [api_url]
+            self.start_url = api_url
```

Comment:
The reasoning here was that there is only one start url (the api link) no matter what. However, maybe this could change in the future, so I changed it back to a list.